docs/source/array-numpy-compatibility.rst

```diff
@@ -0,0 +1,377 @@
+   :obj:`numpy.std`, :obj:`dask.array.std` [#1]_, dask equivalent
+   :obj:`numpy.subtract`, :obj:`dask.array.subtract`, direct (ufunc)
+   :obj:`numpy.sum`, :obj:`dask.array.sum` [#1]_ [#2]_, dask equivalent
+   :obj:`numpy.swapaxes`, :obj:`dask.array.swapaxes`, dask equivalent
+   :obj:`numpy.take`, :obj:`dask.array.take` [#7]_, dask equivalent
+   :obj:`numpy.take_along_axis`, \-
+   :obj:`numpy.tan`, :obj:`dask.array.tan`, direct (ufunc)
+   :obj:`numpy.tanh`, :obj:`dask.array.tanh`, direct (ufunc)
+   :obj:`numpy.tensordot`, :obj:`dask.array.tensordot`, dask equivalent
+   :obj:`numpy.tile`, :obj:`dask.array.tile`, dask equivalent
+   :obj:`numpy.trace`, :obj:`dask.array.trace` [#5]_, dask equivalent
+   :obj:`numpy.transpose`, :obj:`dask.array.transpose`, dask equivalent
+   :obj:`numpy.trapz`, \-
+   :obj:`numpy.tri`, :obj:`dask.array.tri`, dask equivalent
+   :obj:`numpy.tril`, :obj:`dask.array.tril`, dask equivalent
+   :obj:`numpy.tril_indices`, :obj:`dask.array.tril_indices`, dask equivalent
+   :obj:`numpy.tril_indices_from`, :obj:`dask.array.tril_indices_from`, dask equivalent
+   :obj:`numpy.trim_zeros`, \-
+   :obj:`numpy.triu`, :obj:`dask.array.triu`, dask equivalent
+   :obj:`numpy.triu_indices`, :obj:`dask.array.triu_indices`, dask equivalent
+   :obj:`numpy.triu_indices_from`, :obj:`dask.array.triu_indices_from`, dask equivalent
+   :obj:`numpy.true_divide`, :obj:`dask.array.true_divide`, direct (ufunc)
+   :obj:`numpy.trunc`, :obj:`dask.array.trunc`, direct (ufunc)
+   :obj:`numpy.union1d`, :obj:`dask.array.union1d`, dask equivalent
+   :obj:`numpy.unique`, :obj:`dask.array.unique` [#16]_, dask equivalent
+   :obj:`numpy.unpackbits`, \-
+   :obj:`numpy.unravel_index`, :obj:`dask.array.unravel_index`, dask equivalent
+   :obj:`numpy.unwrap`, \-
+   :obj:`numpy.vander`, \-
+   :obj:`numpy.var`, :obj:`dask.array.var` [#1]_, dask equivalent
+   :obj:`numpy.vdot`, :obj:`dask.array.vdot`, dask equivalent
+   :obj:`numpy.vsplit`, \-
+   :obj:`numpy.vstack`, :obj:`dask.array.vstack` [#17]_, dask equivalent
+   :obj:`numpy.where`, :obj:`dask.array.where`, dask equivalent
+   :obj:`numpy.zeros`, :obj:`dask.array.zeros`, dask equivalent
+   :obj:`numpy.zeros_like`, :obj:`dask.array.zeros_like`, dask equivalent
+
+.. rubric:: Footnotes
+
+.. [#1] ``where`` parameter not supported.
+.. [#2] ``initial`` parameter not supported.
+.. [#3] Input must be a dask array.
+.. [#20] ``order`` parameter not supported.
+.. [#4] Sort operations are notoriously difficult to do in parallel. Parallel-friendly alternatives sort the k largest elements.
+.. [#5] ``out`` parameter not supported.
+.. [#6] Use of numpy.matrix is discouraged in NumPy and thus there is no need to add it.
+.. [#7] ``mode`` parameter not supported.
+.. [#8] ``keepdims`` parameter not supported.
+.. [#18] ``fweights``, ``aweights``, ``dtype`` parameters not supported.
+.. [#19] ``like`` parameter not supported. Callable functions not supported.
```

Comment:
Might be missing something but is there a reason why footnotes 20, 18, and 19 are out of the standard order here?

Reply:
... I couldn't find a 'feeling guilty' emojii.... no particular reason but a bit of laziness on my side...  I have reordered the footnotes... :innocent:

Reply:
Ha no worries just making sure I wasn't missing something 😅